src/test/__fixtures/projects/basic-project/test.ts

```diff
@@ -4,6 +4,11 @@
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
+// @ts-nocheck
+import * as BLA from 'not/instrumented'
+
 export function unrelated() {
   return 'hello world'
 }
+
+BLA.functionCall()
```

Comment:
```suggestion
BLA.functionCall()

```

There's a setting you can flip on in vscode to automatically add a new line to the end of files on save. I definitely recommend it as it gets rid of these "no newline" red warning icons at the ends of files.